packages/tags/src/TagsEditor.tsx

```diff
@@ -55,61 +57,83 @@ const styles = {
   }),
 };
 
-const SortablePillHandle = SortableHandle((props: { isDisabled: boolean }) => (
-  <div className={cx(styles.handle, { [styles.pillDisabled]: props.isDisabled })}>
+interface SortablePillHandleProps {
+  ref: (element: HTMLElement | null) => void;
+  listeners: any;
+  isDisabled: boolean;
+}
+
+const SortablePillHandle = ({ ref, listeners, isDisabled }: SortablePillHandleProps) => (
+  <div
+    ref={ref}
+    {...listeners}
+    className={cx(styles.handle, { [styles.pillDisabled]: isDisabled })}
+  >
     <DragIcon variant="muted" />
   </div>
-));
+);
 
 interface SortablePillProps {
+  id: string;
   label: string;
   onRemove: Function;
-  isSortablePillDisabled: boolean;
+  disabled: boolean;
   index: number;
 }
 
-const SortablePill = SortableElement((props: SortablePillProps) => (
-  <Pill
-    testId="tag-editor-pill"
-    className={cx(styles.pill, { [styles.pillDisabled]: props.isSortablePillDisabled })}
-    label={props.label}
-    onClose={() => {
-      if (!props.isSortablePillDisabled) {
-        props.onRemove(props.index);
+const SortablePill = ({ id, label, index, disabled, onRemove }: SortablePillProps) => {
+  const { listeners, setNodeRef, setActivatorNodeRef, transform, transition } = useSortable({ id });
+  const style = {
+    transform: transform ? CSS.Transform.toString({ ...transform, scaleX: 1, scaleY: 1 }) : '',
+    transition,
+  };
+  return (
```

Comment:
Wondering if we could use https://f36.contentful.com/components/drag-handle?

Reply:
Good catch! I will try it.